core/math/math.odin

```diff
@@ -536,11 +723,105 @@ copy_sign :: proc{
 @(require_results) to_degrees_f64   :: proc "contextless" (radians: f64)   -> f64   { return radians * DEG_PER_RAD }
 @(require_results) to_degrees_f64le :: proc "contextless" (radians: f64le) -> f64le { return radians * DEG_PER_RAD }
 @(require_results) to_degrees_f64be :: proc "contextless" (radians: f64be) -> f64be { return radians * DEG_PER_RAD }
+/*
+Convert given input to radians.
+
+**Only accept floats**.
+
+Convertion is implemented by multipication of input and constant `math.RAD_PER_DEG`.
```

Comment:
Don't document the implementation, and if you do, it's a `Note:` or `IMPORTANT:`. This paragraph should be removed.